testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/transaction/mdb/timeout/TransactionTimeoutQueueSetupTask.java

```diff
@@ -83,21 +83,25 @@ public void tearDown(ManagementClient managementClient, String containerId) thro
 
     // /subsystem=messaging-activemq/server=default/address-setting=queue.propertyTimeoutQueue:add(max-delivery-attempts=1)
```

Comment:
would you be so kind and change (or remove) the comment to reflect the changes in the method (aka. `address-setting=queue.` should be `address-setting=jms.queue.`)